contents/resources/dariah-de-publikator/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: DARIAH-DE Publikator Tutorial
+date: 2019-12-01
+authors:
+  - dariah-de
+contributors:
+editors:
+categories:
+  - dariah
+tags:
+  - research-infrastructures
+  - dh
+  - metadata
+  - open-science
+  - sustainability
+  - repositories-and-collections
+  - open-access
+abstract:
+"This tutorial explains the fundamentals of the DARIAH-DE Publikator, the entry point for importing collections and research data into the DARIAH-DE Repository, which allows you to prepare, manage, and finally import your collections into the DARIAH-DE Repository using your favourite internet browser.
+The tutorial contains guides for users as well as technical documentation. The user guides are  conveniently organized into three sections: Collections, Log in with a DARIAH-DE Account or with a Federation Account, and Publishing with the DARIAH-DE Publikator. Every section has examples to explain different aspects of the DARIAH-DE Publikator."
```

Comment:
The abstract should be only one paragraph long, no new lines. This is too long. Just leave the first paragraph. And you can get rid of the quotation marks.

Reply:
Also, the first sentence is a little repetitious (DARIAH-DE Repository is mentioned twice, importing is mentioned twice etc.